pkg/apis/core/v1alpha1/logicalcluster_types.go

```diff
@@ -0,0 +1,175 @@
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	conditionsv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/third_party/conditions/apis/conditions/v1alpha1"
+	"github.com/kcp-dev/kcp/pkg/apis/third_party/conditions/util/conditions"
+)
+
+// LogicalCluster describes the current workspace.
+//
+// +crd
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:subresource:status
+// +kubebuilder:resource:scope=Cluster,categories=kcp
+// +kubebuilder:printcolumn:name="Phase",type=string,JSONPath=`.metadata.labels['tenancy\.kcp\.dev/phase']`,description="The current phase (e.g. Scheduling, Initializing, Ready, Deleting)"
+// +kubebuilder:printcolumn:name="URL",type=string,JSONPath=`.status.URL`,description="URL to access the workspace"
+// +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
+type LogicalCluster struct {
+	v1.TypeMeta `json:",inline"`
+	// +optional
+	v1.ObjectMeta `json:"metadata,omitempty"`
+	// +optional
+	Spec LogicalClusterSpec `json:"spec,omitempty"`
+	// +optional
+	Status LogicalClusterStatus `json:"status,omitempty"`
+}
+
+const (
+	// LogicalClusterName is the name of the LogicalCluster singleton.
+	LogicalClusterName = "this"
+
+	// LogicalClusterFinalizer attached to new ClusterWorkspace (in phase LogicalClusterPhaseScheduling) resources so that we can control
```

Comment:
```suggestion
	// LogicalClusterFinalizer attached to new LogicalCluster (in phase LogicalClusterPhaseScheduling) resources so that we can control
```

Reply:
actually "attached to the owner" of the logical cluster, which normally is a Workspace.

Fixed